test/appearance/appearance_assertion.rb

```diff
@@ -0,0 +1,13 @@
+require 'test/unit'
+
+module AppearanceAssertion
+  include Test::Unit::Assertions
+
+  def assert_same_image(expected_image_path, image_object, delta = 0.01)
+    path = File.expand_path(File.join(__FILE__, '..', expected_image_path))
```

Comment:
I believe you can use `__dir__` here in place of `__FILE__, '..'`.

Reply:
Ah, nice 👍